TACC-Cloud/hazmapper
# task/WG-206: Add layout to map project page

## Overview: ##

Add layout for map project page

The active panel is controlled by a query param.  

I added two "empty" panels to illustrate things and as starting point:
* AssetsPanel which opens the left side panel where the assets listing will go.
* ManageMapProjectModal which opens as a modal

## Related Jira tickets: ##

* [WG-206](https://tacc-main.atlassian.net/browse/WG-206)

## Testing Steps: ##
1. Go to http://localhost:4200/project/84592574-9be5-40cd-8664-1d42c315ef17/ and see if things work
2. Click on Assets to see left sid epanel
3. Click on Manage too see modal

## UI Photos:

![Screenshot 2024-02-15 at 10 10 44 AM](https://github.com/TACC-Cloud/hazmapper/assets/8287580/ea91fec7-e3d4-43e8-b674-a44c190143cc)
![Screenshot 2024-02-15 at 10 10 37 AM](https://github.com/TACC-Cloud/hazmapper/assets/8287580/22ffacdf-a195-4a6b-891d-9ff2c58476e1)
